DXRando/DeusEx/Classes/DeusExCarcass.uc

```diff
@@ -0,0 +1,70 @@
+class DXRDeusExCarcass injects DeusExCarcass;
```

Comment:
we already have a file for this, just reuse `DXRFixes\DeusEx\Classes\Carcass.uc`